zebedee-cms/src/main/java/com/github/onsdigital/zebedee/Init.java

```diff
@@ -19,7 +27,26 @@
 
     @Override
     public void init() {
-        DPLogger.init(new NopConfig());
+
+        try {
+            LogSerialiser serialiser = new JacksonLogSerialiser(true);
```

Comment:
Remove the `true` from the constructor - for prod we don't want it to be formatted